### PR TITLE
fix(test) :Added tests for non collaborators

### PR DESCRIPTION
**Pull request by Raunak1203**
### [ ] What issue/task does this PR references? _[Mandatory]_
https://github.com/fabric8-ui/fabric8-planner/issues/2606

### [ ] Are the tests Included? _[Mandatory]_
Yes

### CODE REVIEW

File: src/tests/specs/collaboratorTest.spec.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  let planner: PlannerPage;
+  let planner1: PlannerPage;
+  let c = new support.Constants();
+
+  beforeAll(async () => {
+    await support.desktopTestSetup();
+    planner = new PlannerPage(browser.baseUrl);
+    await planner.openInBrowser();
+    // // This is necessary since the planner takes time to load on prod/prod-preview
+    await browser.sleep(5000);
```
Comment (jarifibrahim) [inline src/tests/specs/collaboratorTest.spec.ts]: We don't need this. `planner.openInBrowser` will perform login and that's all we need.
You can remove line 16, 17.

File: src/tests/specs/collaboratorTest.spec.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    await planner.ready();
+    let url = await browser.getCurrentUrl()
+    let urlPathName = await browser.executeScript('return document.location.pathname');
+    browser.baseUrl = url.replace(urlPathName,'/rbajpai-test-preview/DO_NOT_DELETE/plan');
+    planner1 = new PlannerPage(browser.baseUrl);
```
Comment (jarifibrahim) [inline src/tests/specs/collaboratorTest.spec.ts]: You can use `planner = new PlannerPage(...)` . 
We won't be using the old value of planner, right?

Comment (Raunak1203) [reply src/tests/specs/collaboratorTest.spec.ts]: No

File: src/tests/specs/collaboratorTest.spec.ts (hunk 1 above)
Comment (Raunak1203) [reply src/tests/specs/collaboratorTest.spec.ts]: Ok Thanks. I'll remove line 16,17

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @Raunak1203 for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


Your plugin configuration is stored in the [file](https://github.com/fabric8-ui/fabric8-planner/c64eb215f4bd0b6f61ac3390d3c817d592e7cfac/test-keeper.yml).

**Comment by Raunak1203** [comment]
[test]

**Comment by Raunak1203** [comment]
Sure @rgarg1

**Comment by Raunak1203** [comment]
[test]

**Pull request merged**